app/src/main/java/com/github/epfl/meili/profile/friends/FriendsListRecyclerAdapter.kt

```diff
@@ -27,8 +27,10 @@ class FriendsListRecyclerAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>
     class FriendViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         private val name: TextView = itemView.findViewById(R.id.friend_name)
 
+        // TODO: fetch and display Friend's info from Firestore: users/{friend.uid}
```

Comment:
I just added this TODO but we can address it in another PR, since it will (probably) take a significant amount of time.

Reply:
The name that is currently displayed is actually the uid.

Reply:
Great! Thank you!